retis/src/core/probe/kernel/bpf/include/retis_context.h

```diff
@@ -37,9 +37,12 @@ struct retis_probe_offsets {
  */
 struct retis_regs {
 #define REG_MAX 12	/* Fexit max, let's use this */
-	u64 reg[REG_MAX];
+/* Extended registers are used to store dynamically extracted arguments. */
+#define EXT_REG_SKB REG_MAX	/* Extended registry for struct sk_buff. */
+#define EXT_REG_MAX EXT_REG_SKB + 1
```

Comment:
Alternatively this is a good candidate for an enum (for easier maintenance):

```c
enum {                                                                          
        REG_MAX = 12,   /* Fexit max, let's use this */                         
        EXT_REG_SKB,                                                            
                                                                                
        __EXT_REG_END,                                                          
        REGS_COUNT = __EXT_REG_END - 1,                                        
};
```

Reply:
This actually makes `EXT_REG_SKB` = 13 (current value is 12) but yep, I get the suggestion.